app/assets/javascripts/angular/components/peopleScreen/peopleScreen.html

```diff
@@ -110,7 +124,8 @@
                     selected="$ctrl.multiSelection[person.id]"
                     person="person" organization-id="$ctrl.org.id"
                     questions="$ctrl.questions"
-                    odd="$odd">
+                    odd="$odd",
+                    show-last-survey="$ctrl.surveyId !== false && $ctrl.surveyId !== undefined">
```

Comment:
Does this work and cover more falsy values?
```suggestion
                    show-last-survey="$ctrl.surveyId">
```

Reply:
well... that just makes me look stupid ... thanks for that.